src/fragments/lib/auth/flutter/signin/60_runtime_auth_flow.mdx

```diff
@@ -0,0 +1,19 @@
+### Switching authentication flow at runtime 
+
+By default, the `authenticationFlowType` value specified in your amplifyconfiguration will be used when authenticating with Cognito. You can change the default behavior at runtime with `CognitoSignInOptions`:
```

Comment:
I think maybe describing what an authenticationFlow means might be helpful! Can we add possibly add that?